industrial_msgs/RobotMode mode
---
int8 rc
